patches/oauth4webapi@2.11.1.patch

```diff
@@ -0,0 +1,44 @@
+diff --git a/build/index.d.ts b/build/index.d.ts
+index c0fff1f253230b19abe8cc864873a573d2ee38a1..36c696c741b03eef7e62806251c074a77f9a8517 100644
+--- a/build/index.d.ts
++++ b/build/index.d.ts
+@@ -920,6 +920,10 @@ export interface DPoPOptions extends CryptoKeyPair {
+      * will be used automatically.
+      */
+     nonce?: string;
++    /**
++     * The DPoP Proof JWT's expiration time in seconds. If not supplied the JWT will have no expiry.
++     */
++    expiresIn?: number;
```

Comment:
Adding in an expiresIn option to dpop because this is required by Bluesky even though the spec doesn't require it.

I've raised this with the bsky team.